src/tests/stdlib.rs

```diff
@@ -22,6 +23,16 @@ fn mkfile() -> (PathBuf, TempDir) {
     (file_path, temp_dir)
 }
 
+fn http_server() {
+    use tiny_http::{Server, Response};
+    
+    let server = Server::http("127.0.0.1:8081").expect("Can't bind to 127.0.0.1:8081");
+    for req in server.incoming_requests() {
+        req.respond(Response::from_string("ok")).expect("Can't respond");
+        break;
+    }
```

Comment:
Whoa! Didn't expect `http_server` in the compiler. Does the `curl` actually download a file?

Reply:
it is used only for tests and that part was implemented by @b1ek.

Because initially I did to download a website but the issue was that what is if the user is not connected to internet or the page is not available?